### ISSUE TITLE
Please Provide Contents from Week 12

### ISSUE THREAD

**Issue by rajat123456** [NAU-ACM/ACM-ICPC-Preparation#4]
From Week 12, the contents are not available, please provide them.
I want to ask 1 thing, that I am also Preparing for ACM ICPC, so can I also contribute to this Project?

**Comment by BedirT** [comment]
hi @rajat123456 sure! Contributions will be judged and accepted with corrections if needed

**Comment by rajat123456** [comment]
Great, Thanks for quick reply :)

**Comment by BedirT** [comment]
@rajat123456 no problem, always 😄. About week 12, and after, we will put it in time. If you have suggestions just send pull request and we will consider.

**Comment by rajat123456** [comment]
Ok sir

**Comment by rajat123456** [comment]
Hello @BedirT  sir, can I add questions from my side also, like if I want to add some more questions on any topic, then can I add them ?

**Comment by BedirT** [comment]
Yeah sure, as long as it's related to topic. Also if the number of questions are a lot we might arrange a new section to keep it clean (like extra questions or so)

**Comment by rajat123456** [comment]
Ok, thanks :)

**Comment by rajat123456** [comment]
@BedirT sir, what is meaning of "By Profile" in Dynamic Programming by Profile in Week 20?
